pull_request_template.md

```diff
@@ -20,4 +20,14 @@
 
 <em>You may erase this after checking them all ;)</em>
 
-- [] CHANGELOG entry added
+- [ ] CHANGELOG entry added.
+- [ ] The latest version was added first in the `CHANGELOG.md`.
+- [ ] PR number added in the CHANGELOG entry.
```

Comment:
Can we add an entry related to maintaining the storybook? Something like:
- [ ] Storybook updated (if necessary).

Reply:
Awesome!